File: algorithm/src/main/java/baekjoon/p09085/Main.java
package baekjoon.p09085;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();

        while (t-- > 0) {
            int n = sc.nextInt();
            int[] inputs = new int[n];

            for (int i = 0; i < inputs.length; i++) {
                inputs[i] = sc.nextInt();
            }

            System.out.println(sum(inputs));
        }
    }

    private static int sum(int[] inputs) {
        int ret = 0;

        for (int input : inputs) {
            ret += input;
        }

        return ret;
    }

}

File: algorithm/src/main/java/baekjoon/p09085/Main.kt
package baekjoon.p09085

fun main(args: Array<String>) {
    repeat(readLine()!!.toInt()) {
        val n = readLine()!!.toInt()
        val inputs = readLine()!!.split(" ").map { input -> input.toInt() }
        println(inputs.sum())
    }
}